spec/consensus/light-client.md

```diff
@@ -193,33 +193,34 @@ We consider the following set-up:
 we will write ```totalVotingPower(V)``` for ```votingpower_in(V,V)```, which returns the total voting power in V.
 We further use the function ```signers(Commit)``` that returns the set of validators that signed the Commit.
 
-**CheckSupport.** The following function checks whether we can trust the header h2 based on header h1 following the trusting period method.
+**CheckSupport.** The following function checks whether we can trust the header h2 based on header h1 following the trusting period method. Time constraint is
+captured by the `hasExpired` function that depends on trusted period (`tp`) and a parameter `Delta` that denotes minimum duration of header so it is
+not considered expired.
 
 ```go
+  func hasExpired(h) {
+    if h.Header.bfttime + tp - Delta < now { // Observation 1
+      return true
+  }
+
+  // basic validation (function `verify`) has already been called on h2
```

Comment:
Also on h1?  I think we should list the functions in the following order:
hasExpired()
verify()
CheckSupport()
Bisection()
Backwards()

Also, add the calling function(s) for Bisection() and Backwards(). 
For all, explain clearly what the return value means, e.g. if verify() returns false it means that the client fails verification and there is no recovery, if CheckSupport() fails it means further bisection is required (?).